### PR TITLE
feat: add combo and dropdown vr tests

### PR DESCRIPTION
**Pull request by smhigley**
Adds both Combobox and Dropdown VR tests (both include essentially the same tests, since the typability of combobox does not affect what would be included in snapshot tests)

### CODE REVIEW

**Review by TristanWatanabe — APPROVED** [review]

**Review by micahgodbolt — APPROVED** [review]

File: apps/vr-tests-react-components/src/stories/Combobox.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
```
[the remark this answers is not in the document]
Comment (behowell) [reply apps/vr-tests-react-components/src/stories/Combobox.stories.tsx:1]: It looks like something broke with the SpinButton tests. Seems to be unrelated, but would be good to figure out what's going on. Did you look into it?

[the remark this answers is not in the document]
Comment (smhigley) [reply apps/vr-tests-react-components/src/stories/Combobox.stories.tsx:1]: Interestingly, yesterday the screenshot tool wouldn't let me access the screenshots. Seems to be working today though 👍

**Review by khmakoto — APPROVED** [review]

File: apps/vr-tests-react-components/src/stories/Combobox.stories.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (smhigley) [reply apps/vr-tests-react-components/src/stories/Combobox.stories.tsx:1]: I couldn't see anything obviously wrong with the SpinButton tests, and the differences seem to be resolved ¯\_(ツ)_/¯

**Pull request merged**